lib/features/exhibition_flow/widgets/organisation_item.dart

```diff
@@ -0,0 +1,103 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app_kids/features/exhibition_flow/widgets/organisation_detail_bottomsheet.dart';
+import 'package:givt_app_kids/features/giving_flow/organisation_details/cubit/organisation_details_cubit.dart';
+import 'package:givt_app_kids/features/recommendation/organisations/models/organisation.dart';
+import 'package:givt_app_kids/features/recommendation/organisations/widgets/organisation_header.dart';
+import 'package:givt_app_kids/helpers/analytics_helper.dart';
+import 'package:givt_app_kids/helpers/app_theme.dart';
+
+class OrganisationItem extends StatelessWidget {
+  const OrganisationItem({
+    super.key,
+    required this.organisation,
+  });
+
+  final Organisation organisation;
+
+  @override
+  Widget build(BuildContext context) {
+    Image image = Image.network(
+      organisation.qrCodeURL,
+      fit: BoxFit.cover,
+    );
+
+    precacheImage(image.image, context);
+
+    return Container(
+      padding: const EdgeInsets.only(bottom: 20),
+      child: InkWell(
+        onTap: () {
+          String generatedMediumId =
+              base64.encode(organisation.namespace.codeUnits);
+          context
+              .read<OrganisationDetailsCubit>()
+              .getOrganisationDetails(generatedMediumId);
+
+          AnalyticsHelper.logEvent(
+            eventName: AmplitudeEvent.charityCardPressed,
+            eventProperties: {
+              AnalyticsHelper.charityNameKey: organisation.name,
+            },
+          );
+
+          showModalBottomSheet<void>(
+            context: context,
+            isScrollControlled: true,
+            backgroundColor: Colors.transparent,
+            builder: (context) => OrganisationDetailBottomSheet(
+              organisation: organisation,
+            ),
+          );
+        },
+        child: Card(
+          elevation: 5,
+          shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(24),
+          ),
+          child: Container(
+            decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(10),
+              color: Colors.white,
+            ),
+            child: Padding(
```

Comment:
Do we need a seperate padding widget or could it be part of the container properties?

Reply:
same here